src/main/java/core/basesyntax/service/impl/ReaderServiceImpl.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.interfaces.ReaderService;
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+
+public class ReaderServiceImpl implements ReaderService {
+    private static final String SOURCE_FILE_PATH = "dataTxt/input.txt";
+
+    @Override
+    public String readFromFile() {
+        File file = new File(SOURCE_FILE_PATH);
```

Comment:
Path need to be defined in method readFromFile() method
```suggestion
    public String readFromFile(String filepath) {
        File file = new File(Path.of(filepath));
```